### PR TITLE
Feature/docker compose enhanced

### PR DESCRIPTION
**Pull request by Luke-1988**

### CODE REVIEW

**Review by Crabzmatic — APPROVED** [review]

**Review by Temppus — APPROVED** [review]

File: sf-docker/.env.sfac [hunk 1]
```diff
@@ -0,0 +1 @@
+#FilterConfiguration__SpoofCheckConfiguration__Enabled=true
```
Comment (blaskoa) [inline sf-docker/.env.sfac:1]: do we really want this enabled by default?

File: sf-docker/sf_dependencies/docker-compose.yml [hunk 2]
```diff
@@ -8,13 +8,21 @@ services:
     environment: 
       - ACCEPT_EULA=Y
       - SA_PASSWORD=Test1234
+    restart: unless-stopped
```
Comment (blaskoa) [inline sf-docker/sf_dependencies/docker-compose.yml:11]: If we are doing this for the dependencies, we could also set it up for SF

File: sf-docker/docker-compose.yml [hunk 3]
```diff
@@ -334,6 +334,7 @@ services:
       - RabbitMq__Password=${RabbitMQ__Password}
       - RabbitMq__Port=${RabbitMQ__Port}
       - Kestrel__EndPoints__Http__Url=${Hosting__Host}:${Hosting__Port}
+    env_file: .env.sfac
```
Comment (marek-nereca) [inline sf-docker/docker-compose.yml:337]: same change should be added to jetson-docker-compose.yml
what about variables like RabbitMQ__Password which are defined in .env file? are variables from .env file merged with variables from .env.sfac?

File: sf-docker/.env.sfac (hunk 1 above)
Comment (Luke-1988) [reply sf-docker/.env.sfac:1]: it is commented - just a sample how it should be composed

File: sf-docker/docker-compose.yml (hunk 3 above)
Comment (Luke-1988) [reply sf-docker/docker-compose.yml:337]: I think yes, I am using it this way on SRV-BA-6D and it is working

File: sf-docker/sf_dependencies/docker-compose.yml (hunk 2 above)
Comment (Luke-1988) [reply sf-docker/sf_dependencies/docker-compose.yml:11]: added

**Pull request merged**